test/descartes_path_service_capability_test.cpp

```diff
@@ -66,11 +75,72 @@ class MoveGroupDescartesPathServiceTest : public ::testing::Test
   ros::NodeHandle nh_;
 };  // class MoveGroupDescartesPathServiceTest
 
-TEST_F(MoveGroupDescartesPathServiceTest, NameOfTest)
+TEST_F(MoveGroupDescartesPathServiceTest, TestDescartesCapabilityAvailable)
+{
+  try
+  {
+    pluginlib::ClassLoader<move_group::MoveGroupCapability> capability_plugin_loader("moveit_ros_move_group",
+                                                                                     "move_group::MoveGroupCapability");
+    // Test a move_group default plugin
+    EXPECT_TRUE(capability_plugin_loader.isClassAvailable("move_group/MoveGroupCartesianPathService"));
+    // Test the Descartes capability
+    EXPECT_TRUE(capability_plugin_loader.isClassAvailable("descartes_capability/MoveGroupDescartesPathService"));
+  }
+  catch (pluginlib::PluginlibException& ex)
+  {
+    std::cerr << "Exception while creating plugin loader for move_group capabilities: " << ex.what() << std::endl;
+    EXPECT_TRUE(false);
+  }
+  EXPECT_TRUE(true);
```

Comment:
Is this one necessary?

Reply:
The tests have to output something otherwise travis complains that the tests don't have any output

This particular one is not essential so I can remove it